views/confirm.hbs

```diff
@@ -18,16 +25,22 @@
     <!-- SCRIPTS -->
     <script src='/js/confirm.js'></script>
 </head>
+
 <body>
     <!-- <div class='wrapper'> -->
     <div class='confirm'>
-        <h2>Please confirm your account!</h2>
-        <p>A confirmation has been sent to your email address. Please check your inbox and click on the unique link to verify your account.</p>
+        <div class="thankyou">
+            <h1>Thank you for signing up!</h1>
+        </div>
+        <p>A confirmation has been sent to your email address. Please check your inbox and click on the unique link to
+            verify your account.</p>
+        <hr style="border:2px solid #50b5dd; margin:15px 60px" />
```

Comment:
Idk if you intended for the `<hr>` to have a stroke all the way around it. Generally people just apply `border-bottom` to them, but if that's what you were going for, then we're good